apps/aecore/lib/aecore/peers/sync.ex

```diff
@@ -20,7 +20,7 @@ defmodule Aecore.Peers.Sync do
   def start_link(_args) do
     GenServer.start_link(
       __MODULE__,
-      %{peer_blocks: %{}, peer_block_tasks: %{}, chain_sync_running: false},
+      %{peer_blocks: %{}, peer_block_tasks: %{}, chain_sync_status: false},
```

Comment:
why is this renamed? status true|false is semantic incorrect, running true|false was correct

Reply:
I renamed it to running in a previous issue, and now I thought that it was maybe a bit confusing. I'll change it.

Reply:
if you want to use status as name, use atoms as identifier

Reply:
No, I'll rename it as you suggested. The semantic name does make more sense.

Reply:
I'm just saying status could be semantical if it is not boolean, but atom

Reply:
Yes, but then in the places where the status is checked with `if statement` it should be changed with `case`, and there is no reason for that, if I can just return the naming.